README.md

```diff
@@ -66,7 +66,7 @@ release.
 #### Container Image
 
 The latest container image can be found at:
-* `quay.io/coreos/kube-state-metrics:v1.1.0`
+* `quay.io/coreos/kube-state-metrics:v1.2.0-rc.0`
 * `k8s.gcr.io/kube-state-metrics:v1.1.0`
```

Comment:
Actually this is just an example demonstrating that the image for kube-state-metrics is available both on `quay.io` and `k8s.gcr.io`. So, personally i think the tag version can be stale. However if we need to keep those tags in sync with the release, we should update both for `quay.io` and `k8s.gcr.io`. 

@zouyee

Reply:
For now, ksm image in k8s.gcr.io has not been updated.

Reply:
Yep. But it will be updated by @loburm later. I think we do not need another PR to just update this as @loburm has acknowledged the updating in https://github.com/kubernetes/kube-state-metrics/pull/335#issuecomment-355589236. :)